src/System.Diagnostics.Process/tests/ProcessThreadTests.cs

```diff
@@ -78,10 +77,10 @@ public void TestStartTimeProperty()
                     // Time in unix, is measured in jiffies, which is incremented by one for every timer interrupt since the boot time.
                     // Thus, because there are HZ timer interrupts in a second, there are HZ jiffies in a second. Hence 1\HZ, will
                     // be the resolution of system timer. The lowest value of HZ on unix is 100, hence the timer resolution is 10 ms.
-                    // Allowing for error in 10 ms.
-                    long tenMSTicks = new TimeSpan(0, 0, 0, 0, 10).Ticks;
-                    long beforeTicks = timeBeforeCreatingProcess.Ticks - tenMSTicks;
-                    long afterTicks = DateTime.UtcNow.Ticks + tenMSTicks;
+                    // Allowing for error in 10 ms, on windows 15ms.
```

Comment:
I'm not sure what this comment means... where are we still using the 10ms number in the code?